src/sentry/uptime/models.py

```diff
@@ -73,6 +76,7 @@ class Meta:
                 "interval_seconds",
                 "timeout_ms",
                 "method",
+                "trace_sampling",
```

Comment:
Because you have a custom migration body this change isn't part of the DDL. Modifying existing constraints is challenging as postgres doesn't have an alter constraint command. Instead we have to make a new constraint and drop the old one. I recommend adding the new constraint first, and then removing the old one to ensure there are no constraint gaps.

Reply:
ah gotcha. thanks. will follow the procedure that was done in this PR previously on this model. https://github.com/getsentry/sentry/pull/78787

Reply:
I notice in that PR we did not follow procedure and had a constraint gap. @evanpurkhiser @wedamija should we worry about this now?